From d693e2b316ecb48c6803dc65ae994ca7fff6532c Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Sun, 8 Sep 2019 00:21:17 +0200
Subject: [PATCH 4/4] End of line

---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 493fb42..a215997 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -11,7 +11,7 @@ Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@  (<< 0.7.0-2)
-Breaks: @(', '.join(Replaces))@  (<< 0.7.0-2)@[end if]@
+Breaks: @(', '.join(Replaces))@  (<< 0.7.0-2)@\n@[end if]@
 Description: @(Description)
 
 Package: @(Package.replace('ros-ign','ros1-ign'))
-- 
2.17.1

